source/help/getting-started/searching.rst

```diff
@@ -0,0 +1,115 @@
+Searching
```

Comment:
Not sure if it's just an issue in github, but this doc is rendering in raw rst format.